### ISSUE TITLE
Getting PHP notices when auto-saving draft

### ISSUE THREAD

**Issue by psaikali** [AdvancedCustomFields/acf#582]
I'm getting the following PHP notice in the debug.log file when WordPress auto-saves a (CPT) post:

```
PHP Notice:  Undefined variable: item in /app/wp-content/plugins/advanced-custom-fields-pro/includes/ajax/class-acf-ajax-check-screen.php on line 60
```

<img width="679" alt="Screen 2021-11-23 à 10 46 56" src="https://user-images.githubusercontent.com/1606655/143002509-b8d858b8-2edd-4f94-b335-d25aff4d5744.png">

**Comment by lgladdy** [comment]
@psaikali Thanks for the report! We'll get this fixed up!

**Comment by lgladdy** [comment]
Hey @psaikali. This was fixed in today's 5.11.2 release. Thanks again for your report!
